raiden/raiden_service.py

```diff
@@ -366,8 +378,12 @@ def get_new_events(self):
 
         return event_list
 
-    def open(self, token_address, partner_address,
-             settle_timeout=None, reveal_timeout=None):
+    def open(
+            self,
+            token_address,
+            partner_address,
+            settle_timeout=None,
+            reveal_timeout=None):
```

Comment:
I think that yes moving this to a separate issue makes sense since it needs some work and is well defined enough to be its own small issue.